packages/cinterop/src/jvm/kotlin/io/realm/kotlin/internal/interop/gc/NativeContext.kt

```diff
@@ -25,8 +25,11 @@ object NativeContext {
     private val finalizingThread = Thread(FinalizerRunnable(referenceQueue))
 
     init {
-        finalizingThread.name = "RealmFinalizingDaemon"
-        finalizingThread.start()
+        finalizingThread.run {
+            name = "RealmFinalizingDaemon"
+            isDaemon = true
+            start()
```

Comment:
Dont we manually setup a priority? as we do in `CoroutineUtilsSharedJvm`? If not set it will default to the priority of the thread it was created from:

>When code running in some thread creates a new Thread object, the new thread has its priority initially set equal to the priority of the creating thread, and is a daemon thread if and only if the creating thread is a daemon.

We might like to have some control on it.